lms/validation/authentication/_bearer_token.py

```diff
@@ -82,7 +82,7 @@ def authorization_param(self, lti_user):
         """
         return self.dump(lti_user)["authorization"]
 
-    def lti_user(self):
+    def lti_user(self, location):
```

Comment:
`BearerTokenSchema.lti_user()` used to parse three locations at once by doing `self.parse(locations=["headers", "querystring", "form"])`. Webargs no longer supports that so I added a required `location` arg to `lti_user()` and the calling code now calls it three times with `"headers"`, `"querystring"`, and `"form"` and makes its own decision about what to do with the three `LTIUser`'s or `ValidationError`'s.